packages/block-library/src/navigation/index.php

```diff
@@ -148,10 +162,29 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 		)
 	);
 
+	$modal_unique_id = uniqid();
+
+	// Determine whether or not navigation elements should be wrapped in the markup required to make it responsive.
+	$responsive_container_markup = isset( $attributes['isResponsive'] ) && true === $attributes['isResponsive'] ? sprintf(
```

Comment:
Nitpick: It feels to me that having an early return for non responsive would be a bit more readable, like 
`if ( ! isset( $attributes['isResponsive'] ) ||  ! $attributes['isResponsive'] ) { return non responsive markup... } `

Not a blocks though of course..

Reply:
100% agreed, I'll refactor this.